practices/smart_goals.md

```diff
@@ -0,0 +1,123 @@
+
+![](/files/smart3.png) 
+
+## What Are SMART Goals?
+
+**SMART** is an acronym that you can use to guide your goal setting. Using SMART Goals sets boundaries and defines the steps you will need to take, resources necessary to get there, and milestones that indicate progress along the way. To make sure your goals are clear and reachable, each one should be:
+
+- **S**pecific (simple, sensible, significant).
+- **M**easurable (meaningful, motivating).
+- **A**ttainable (agreed, achievable).
+- **R**elevant (reasonable, realistic and resourced, results-based).
+- **T**ime bound (time-based, time limited, time/cost limited, timely, time-sensitive).
+
+### Specific
+
+Specify your goal to be able to focus your efforts. The following questions can help you: 
+
+- What do I want to accomplish?
+- Why is this goal important?
+- Who is involved?
+- Where is it located?
+- Which resources or limits are involved?
+
+### Measurable
+
+With measurable goals set, you can track your progress and stay motivated. Clearly set deadlines will help you to assess your progress. 
+A measurable goal should address questions such as:
+
+- How much?
+- How many?
+- How will I know when it is accomplished?
+
+### Attainable
+
+Your goal also needs to be realistic and attainable to be successful. When you set an achievable goal, you may be able to identify previously overlooked opportunities or resources that can bring you closer to it.
+To set an attainable goal, you will have to deal with the following questions:
+
+- How can I accomplish this goal?
+- How realistic is the goal, based on other constraints, such as financial factors?
+
+**NOTE**: Do not set goals that depend on someone else. Only goals that depend on you, your effort, and your skills are convenient ones. 
+
+### Relevant
+
+Ensure that your goal matters to you, and is not in conflict with your other relevant goals. So, make sure that your plans drive everyone forward, but that you're still responsible for achieving your own goal.
+In order to set a relevant goal, answer these questions (answers should be “Yes”): 
+
+- Does this seem worthwhile?
+- Is this the right time?
+- Does this match my other efforts?
+- Am I the right person to reach this goal?
+- Is it applicable in the current (socio-)economic environment?
+
+### Time-bound
+
+What is your goal time-frame? An end-date can provide motivation and help you to prioritize and prevent everyday tasks from taking priority over your longer-term goals.
+To set a time-bound goal, you will deal with the following questions:
+
+- When?
+- What can I do six months from now?
+- What can I do six weeks from now?
+- What can I do today?
+
+## How to Start With SMART Goals
+
+There are various SMART Goals worksheets, templates, and maps that can help you with the goals setting process. 
+
+![](/files/smart_templ1b.png)
+
```

Comment:
Alternatively, you could move the whole passage starting with ### Specific since on the inside of the paragraphs you show how to use this methodology (which question should I ask and so on and so on).

Reply:
Moved